src/SimpleThings/EntityAudit/AuditReader.php

```diff
@@ -153,215 +53,63 @@ public function __construct(EntityManager $em, AuditConfiguration $config, Metad
     }
 
     /**
-     * @return \Doctrine\DBAL\Connection
-     */
-    public function getConnection()
-    {
-        return $this->em->getConnection();
-    }
-
-    /**
-     * @return AuditConfiguration
-     */
-    public function getConfiguration()
-    {
-        return $this->config;
-    }
-
-    /**
-     * Clears entity cache. Call this if you are fetching subsequent revisions using same AuditManager.
-     */
-    public function clearEntityCache()
-    {
-        $this->entityCache = array();
-    }
-
-    /**
      * Find a class at the specific revision.
      *
      * This method does not require the revision to be exact but it also searches for an earlier revision
-     * of this entity and always returns the latest revision below or equal the given revision. Commonly, it
-     * returns last revision INCLUDING "DEL" revision. If you want to throw exception instead, set
-     * $threatDeletionAsException to true.
+     * of this entity and always returns the latest revision below or equal the given revision
      *
      * @param string $className
      * @param mixed $id
      * @param int $revision
-     * @param array $options
      * @return object
-     * @throws DeletedException
-     * @throws NoRevisionFoundException
-     * @throws NotAuditedException
-     * @throws \Doctrine\DBAL\DBALException
      */
-    public function find($className, $id, $revision, array $options = array())
+    public function find($className, $id, $revision)
     {
-        $options = array_merge(
-            array(
-                'threatDeletionsAsExceptions' => false
-            )
-            , $options
-        );
-
         if (!$this->metadataFactory->isAudited($className)) {
-            throw new NotAuditedException($className);
+            throw AuditException::notAudited($className);
```

Comment:
ok I will fix this tomorrow.

Regards,

Vasil

On Fri, Oct 16, 2015 at 6:37 PM, David Badura <notifications@github.com>
wrote:

> In src/SimpleThings/EntityAudit/AuditReader.php
> <https://github.com/simplethings/EntityAudit/pull/139#discussion_r42256206>
> :
>
> >          if (!$this->metadataFactory->isAudited($className)) {
> > -            throw new NotAuditedException($className);
> > +            throw AuditException::notAudited($className);
>
> No, NotAuditedException isn't the same name as AuditException [image:
> :wink:] Do you forget a use statement or add the class?
>
> The builds are failed, see
> https://travis-ci.org/simplethings/EntityAudit/builds/69906870
>
> PHP Fatal error:  Class 'SimpleThings\EntityAudit\AuditException' not found in /home/travis/build/simplethings/EntityAudit/src/SimpleThings/EntityAudit/AuditReader.php on line 411
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/simplethings/EntityAudit/pull/139/files#r42256206>.
>